### PR TITLE
1/gatsby plugin gdpr

### PR DESCRIPTION
**Pull request by hagoptaminian**

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -74,20 +74,18 @@ export default () => {
...
-          <a
-            href="https://graasp.eu/views/privacy_policy.client.html"
-            title="Privacy Policy"
-          >
+          <a href="/privacy" title="Privacy Policy">
             Privacy Policy
           </a>
+          {' | '}
+          <a href="/disclaimer" title="Legal Disclaimer">
+            Legal Disclaimer
```
Comment (juancarlosfarah) [inline src/components/footer.js:87]: Shouldn't these titles go in `t()` tags?

File: src/pages/disclaimer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Layout from '../components/layout';
+import ContentContainer from '../components/content-container';
+
+const Disclaimer = () => {
+  const heading = 'Legal Disclaimer';
+
+  const content = (
+    <p>
+      Graasp Insights (the &lsquo;software&rsquo;) and its source code are
```
Comment (juancarlosfarah) [inline src/pages/disclaimer.js:10]: Not sure if these should also go in `t()` tags. If it's too complex, let's make a different Issue and address it separately (if French is needed, for instance).

File: src/pages/privacy.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import React from 'react';
+import Layout from '../components/layout';
+import ContentContainer from '../components/content-container';
+
+const PrivacyPolicy = () => {
+  const heading = 'Privacy Policy';
+
+  const content = (
+    <p>
+      Graasp Insights is developed and maintained by the{' '}
```
Comment (juancarlosfarah) [inline src/pages/privacy.js:11]: Same as above, but it seems that these longer texts will be complicated to include with `t()` so let's leave it for a different PR.

**Review by juancarlosfarah — APPROVED** [review]
Works as expected! Just some comments about translations with `t`, but let's leave it for another PR.

File: src/components/footer.js (hunk 1 above)
Comment (hagoptaminian) [reply src/components/footer.js:87]: we don't have i18next set up for either of -content-insights or -content-desktop
